.circleci/jobs/GradleJob.pkl

```diff
@@ -37,15 +41,6 @@ job: Config.Job = new {
   steps {
     "checkout"
     ...module.steps
-    new Config.RunStep {
-      // find all test results and write them to the home dir
-      name = "Gather test results"
-      command = """
-        mkdir ~/test-results/
-        find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} ~/test-results/ \\;
-        """
-      `when` = "always"
-    }
```

Comment:
This step breaks on Windows.

Instead of trying to figure out how to write a Windows version, I added some logic to specify the test reports dir (see line 29).